lk2nd/lk2nd-device.c

```diff
@@ -183,6 +183,21 @@ static bool match_string(const char *s, const char *match, size_t len)
 	return strncmp(s, match, len + 1) == 0;
 }
 
+static bool match_strings(const char *s, const char *match, size_t len)
+{
+	size_t entry_len, used_len = 0;
+
+	while (used_len < len) {
+		entry_len = strnlen(match + used_len, len - used_len);
+		if (match_string(s, match + used_len, entry_len))
```

Comment:
At first I thought is should be
```diff
-if (match_string(s, match + used_len, entry_len))
+if (match_string(s, match + used_len, entry_len - used_len))
```
but I guess match_string can handle string length by otself?